### ISSUE TITLE
Command not found: add-zsh-hook

### ISSUE THREAD

**Issue by djornada** [wbinglee/zsh-wakatime#8]
Hello,
I'm having a problem when I open the terminal I get the following mesage:

```zsh
~/.oh-my-zsh/custom/plugins/zsh-wakatime/zsh-wakatime.plugin.zsh:19: command not found: add-zsh-hook
zsh --version
zsh 5.2 (x86_64-unknown-linux-gnu)
```
Can you help me fix it?

**Comment by alanhamlett** [comment]
Maybe this is the fix?
#6

**Comment by djornada** [comment]
Seems was fixed, but i'm not sure about tracking the URL is not accessible

**Comment by wbinglee** [comment]
@djornada  Do you mean you cannot access https://wakatime.com/project/Terminal  ?

**Comment by mrtnrdl** [comment]
Thank's @alanhamlett, that fixed it for me.

**Comment by DevinXian** [comment]
Thank's @alanhamlett, saved my time ^_^

**Comment by djornada** [comment]
Hello, @wbinglee i'm using Arch Linux, and yes I can't acess https://wakatime.com/project/Terminal

**Comment by wbinglee** [comment]
I have merged the PR #6 . Let me know if any problem.

**Comment by wbinglee** [comment]
@djornada  BTW, do you still cannot see any data from zsh?   which wakatime version do you use? `wakatime --version`

**Comment by djornada** [comment]
Hi @wbinglee 
```zsh
wakatime --version
6.0.1
```

**Comment by wbinglee** [comment]
@djornada  I have merged the fix for waka 6.0+. You can pull the change and try again (in a new terminal).

**Comment by djornada** [comment]
@wbinglee Thanks, it's working now
